### PR TITLE
storage service starts using version.hpp

### PR DESCRIPTION
**Pull request by Jinming-Hu**
closes https://github.com/Azure/azure-sdk-for-cpp/issues/708
closes https://github.com/Azure/azure-sdk-for-cpp/issues/543

### CODE REVIEW

**Review by RickWinter — APPROVED** [review]

File: sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/version.hpp [hunk 1]
```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
```
Comment (katmsft) [inline sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/version.hpp:5]: What's this tag for?

Comment (Jinming-Hu) [reply sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/version.hpp:5]: copied from https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/core/azure-core/inc/azure/core/version.hpp

**Comment by katmsft** [comment]
Please also provide a list of files that need to be changed during a version upgrade in OneNote's release steps.

**Review by katmsft — APPROVED** [review]

**Pull request merged**